static-assets/scripts/admin.js

```diff
@@ -423,6 +423,7 @@
       audit.allTimeZones = moment.tz.names();
       audit.sort = 'date';
       $scope.originValues = [$translate.instant('admin.audit.ALL_ORIGINS'), 'API', 'GIT'];
+      $scope.sites = CrafterCMSNext.system.store.getState().sites;
```

Comment:
I think the API now gives you the site name too, so we don't need to keep a separate site lookup, no?
If we do, best if we prep for this being async.